scripts/dockerimages/scripts/console.sh

```diff
@@ -82,6 +82,14 @@ if ! grep -q '^UseDNS no' /etc/ssh/sshd_config; then
     echo "UseDNS no" >> /etc/ssh/sshd_config
 fi
 
+if ! grep -q '^PermitRootLogin no' /etc/ssh/sshd_config; then
+    echo "PermitRootLogin no" >> /etc/ssh/sshd_config
+fi
+
+if ! grep -q '^AllowUsers rancher' /etc/ssh/sshd_config; then
```

Comment:
We should not limit access to the user `rancher`. One might add additional user accounts via `cloud-config`, which then would be denied to login.